cirq/protocols/json_serialization.py

```diff
@@ -157,7 +158,7 @@ def two_qubit_matrix_gate(matrix):
                 cirq.devices.unconstrained_device._UnconstrainedDevice,
                 'VirtualTag': cirq.VirtualTag,
                 'WaitGate': cirq.WaitGate,
-                '_QubitAsQid': raw_types._QubitAsQid,
+                '_QubitAsQid': cirq.NamedQid,
```

Comment:
This is incorrect.  Instead of changing `_QubitAsQid` to `NamedQid`, change the test to another type of qubit that doesn't have a qudit variant.